packages/components/src/components/Icon/index.tsx

```diff
@@ -125,7 +133,7 @@ const Icon = React.forwardRef(
                     beforeInjection={svg => {
                         svg.setAttribute('width', `${size}px`);
                         svg.setAttribute('height', `${size}px`);
-                        svg.setAttribute('fill', defaultColor);
+                        // svg.setAttribute('fill', defaultColor);
```

Comment:
The comment should not be there obv. Removed it because I am coloring from the SvgWrapper now since there were issues with SVGs not being all the same code-wise. Some needed `fill`, some `stroke` to be colored. For now fixed with the `isHollow` prop.

Reply:
Thanks, if you leaving commented parts of code, leave also comment why.

Reply:
I simply forgot to remove it after testing :( Will check the PR myself first next time.

Reply:
I strongly recommend to do that, checking in different environment (on Github) will help you see lot of problems you could have missed. I am doing this and I always find something that I missed.